config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
+  get '/ride_dashboard', to: "ride/sessions#new"
```

Comment:
Does anyone think we should be nesting these resources?

Reply:
That was my initial thought too, but if there's a reason for not nesting, that's fine.